From 37877d5c4eba6ec9c03678d2b4faaaa87011bb8d Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 14 Dec 2016 12:06:49 -0800
Subject: [PATCH] TOOLS-1636 updates-imgadm et al with node v4 will not flush
 stdout on exit Reviewed by: Julien Gilli <julien.gilli@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index f72a187..69bd22e 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "dependencies": {
     "manta": "3.1.3",
     "smartdc": "7.3.0",
-    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#38332b1"
+    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.3"
   },
   "engines": {
     "node": ">=0.8"
-- 
2.21.0

